migrations/20160623184725_set_protected_roles.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+use Phinx\Migration\AbstractMigration;
+
+class SetProtectedRoles extends AbstractMigration
+{
+    
+    /**
+     * Migrate Up.
+     */
+    public function up()
+    {
+        $this->execute("UPDATE roles
+        	SET protected = 1
+            WHERE
+                name = 'user' OR name = 'admin'
+            ");
```

Comment:
Could this migration reinstate the user and admin roles if they don't exist too?